calculator.js

```diff
@@ -1 +1,180 @@
-console.log('start')
\ No newline at end of file
+const numbers = document.querySelectorAll('.number')
+const mainResult = document.querySelector('.result')
+const operators = document.querySelectorAll('.operator')
+const equal = document.querySelectorAll('.equal')
+const deleteEqual = document.querySelectorAll('.deleteEqual')
+const dot = document.querySelectorAll('.dot')
+const deleteLast = document.querySelectorAll('.deleteLast')
+
+let result = ''
```

Comment:
Why result is not 0? The app is displaying 0 at the begining it is not displaying the result?

The general advice is that you should:
- when user takes action, handle should trigger your logic (function)
- your logic should update the state
- then render the whole state at the end as the sate should be 1 source of truth

P.S. For you the state are ofc there lets defined here. :)

Reply:
fixed ? :P